src/ScrollyVideo.js

```diff
@@ -62,8 +63,8 @@ class ScrollyVideo {
     this.useWebCodecs = useWebCodecs;
     this.cover = cover;
     this.sticky = sticky;
-    this.full = full;
```

Comment:
why remove this?

Reply:
`this.full` is not being used anywhere in this class

`full` arg in constructor is used on instantiation only:
<img width="426" alt="CleanShot 2024-05-02 at 08 22 02@2x" src="https://github.com/dkaoster/scrolly-video/assets/21989873/145645bc-4c9f-4fa5-b267-d1ccdc67d312">